### PR TITLE
remove os-net-mapping from nic_configs

### PR DESCRIPTION
**Pull request by asyedham**

### CODE REVIEW

**Comment by asyedham** [comment]
I have tested it for OSP13, yet to test for OSP16.

File: ci/all_osp13.yml [hunk 1]
```diff
@@ -31,29 +31,55 @@ alias:
...
-      r630: 
-        interfaces: [enp1s0f0, enp1s0f1, enp1s0f2, enp1s0f3]
-      r730xd:
-        interfaces: [enp1s0f0, enp1s0f1, enp130s0f0, enp130s0f1]
-      r930:
-        interfaces: [enp1s0f0, enp1s0f1, enp10s0f0, enp10s0f1]
+  rhel7:
+    machine_types:
+      supermicro:
+        1029p:
```
Comment (venkataanil) [inline ci/all_osp13.yml]: Can you please organise these like below (please remove interfaces key and in ansible tasks also)
http://paste.openstack.org/show/792800/

File: prepare_nic_configs.yml [hunk 2]
```diff
@@ -20,35 +20,44 @@
...
+            ifaces: "{{ item.value.rhel7_interfaces }}"
+          with_dict: "{{ scale.machine_types.supermicro }}"
+          when: item.key == machine_type and (osp_release <= 13) and machine_type in scale.machine_types.supermicro
 
         - name: set interfaces
           set_fact:
-            ifaces: "{{ item.value.interfaces }}"
-          with_dict: "{{ scale.rhel8.machine_types.supermicro }}"
-          when: item.key == machine_type and (osp_release > 13) and machine_type in scale.rhel8.machine_types.supermicro
+            ifaces: "{{ item.value.rhel8_interfaces }}"
```
Comment (venkataanil) [inline prepare_nic_configs.yml]: Try ternary operator here
- name: set interfaces
  set_fact:
      ifaces: "{{ (osp_release|int > 13)| ternary(item.value.rhel8_interfaces, item.value.rhel7_interfaces) }}"
  with_dict: "{{ scale.machine_types.supermicro }}"
  when: item.key == machine_type and machine_type in scale.machine_types.supermicro

With this you can merge two tasks into one.

Comment (asyedham) [reply prepare_nic_configs.yml]: Fixed

**Pull request merged**